applications/asset_tracker/prj.conf

```diff
@@ -24,6 +24,7 @@ CONFIG_MQTT_MAX_PACKET_LENGTH=2048
 # LTE link control
 CONFIG_POWER_OPTIMIZATION_ENABLE=n
 CONFIG_LTE_LINK_CONTROL=y
+CONFIG_LTE_NETWORK_MODE_LTE_M_GPS=y
```

Comment:
Thanks for this change! Could you also add it to `prj_nrf9160_pca20035ns.conf` for Thingy:91?

Reply:
Thank you. Sorry to missed that and I'll add it.

Reply:
> Thanks for this change! Could you also add it to `prj_nrf9160_pca20035ns.conf` for Thingy:91?

Does this enable GPS when PSM is not available on Thingy:91?

Reply:
This is only to configure modem mode to be LTE-M + GPS (which is the default mode before PR). The assert_tracker enable GPS by long press the button on Thingy:91, while PSM mode make sure GPS acquisition actually runs or not.